packages/desktop-client/src/components/ManageRules.js

```diff
@@ -588,20 +588,34 @@ export default function ManageRules({
   }, []);
 
   function onCreateRule() {
+    let rule = {
+      id: null,
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏why do we need to set `id` here? What happens if we do not set it?

Reply:
I first tried to remove `id: null and stage:null` which threw on save, but that is related to stage. I have removed as per the comment